### ISSUE TITLE
More routers in system

### ISSUE THREAD

**Issue by MiG-41** [Kajkac/ZTE-MC-Home-assistant-repo#17]
I have for tests MC889A Pro , and till now i use it replacing my MC7010 ( I did it with simplest way , give to MC889A Pro same IP and same password ,with MC7010 had). Integration starts read from new device without any changing. 
Now i decided to connect it parallel ( using mwan3 on OpenWRT router in balanced mode) , it working properly. ( I changed the IP on MC889A Pro therefore). 
So decided to Add HUB in integration on HA , put credentials ( pasword same , ip different) , but it is not possible to do it, it shows error.
In log i see this:
![image](https://github.com/user-attachments/assets/3bacddd3-c9b2-4107-aaf5-7a07171b661f)

I can see some entities ,but they are unknown:
![image](https://github.com/user-attachments/assets/85eea82c-931c-4436-a28a-e36a3987e936)

And next thing, this new entities are vary from first device only with "..._2" ,maybe there is some way to name entities with starts i.e. by given name ,so there wont be this case.

**Comment by MiG-41** [comment]
Turns out ,that choosing from menu MC889 instead MC889A reads entities properly from MC889A PRO ( the same case with MC7010 , also needs to be choosen MC889).... Is there some difference in finding entities or ...? 
![image](https://github.com/user-attachments/assets/b4616d82-c45e-4f2a-878b-c7933d591670)

Maybe it is @derkrasseleo issue in #15  ....

So it is generally no problem to have two integration ( only proper config have to be selected).

Only case with is left is the name of entities.... Not to have "_2" at the end , but be able to figure out with one is for with routers ,adding a name , or IP , or....

**Comment by Kajkac** [comment]
will try to figure what here deal is... MC888A and MC889A should be the username models , but mc889a pro is probably different model then mc889 , i have mc888 , mc889 mc801 and mc888 Ultra, all of them dont have usernames , also only mc801 have http and https , when other ones only https, and yeah firwares are shit, no generic firmware per se, only provider firmwares. 

Problem is  on all of this new ones that they have locked bootloaders / aka secure boot , so no modding for now possible to be able to flash some generic firmware (i do have one mc801 with ZTE logo and zte generic firmware but that was the only model i got with no branding)

So every model have different options and features, ca combos and stuff but all of them are the same device an OS (GOFORM)/android inside

**Comment by derkrasseleo** [comment]
I agree, I find it really frustrating as well that it's not easier to get basic information from a modem..

> when on my integration you have const.py where i try to map sensors , and stuff like that per model, but hell so many models and providers i cant get that to be up to date with all versions in the wild.

Maybe you could read this from config files for each router/modem (json, yaml etc.) where the sensors are stored  in a specific format? This way people could contribute the specific sensors their device has..

**Comment by Kajkac** [comment]
Basicly integration gets all the sensor, some are emoty some not, but lot of this is trials and errors to see what fe have what.
